github/AuthenticatedUser.py

```diff
@@ -373,7 +374,9 @@ def add_to_emails(self, *emails):
         :rtype: None
         """
         assert all(isinstance(element, str) for element in emails), emails
-        post_parameters = emails
+        post_parameters = {
+            "emails": emails
+        }
```

Comment:
Let me guess, GitHub also requires the emails key now. Bah. (I'm annoyed at GitHub changing the API without notice, rather than this change.)

Reply:
I think so from https://docs.github.com/en/rest/reference/users#add-an-email-address-for-the-authenticated-user